stuffListApi/public/controllers/controller.js

```diff
@@ -0,0 +1,176 @@
+var app = angular.module('myStuffApp', []);
```

Comment:
we can use also something like: 
`'views/andTheViewsInside'`
`'services/andTheServicesInside'`
`'filters/andTheFiltersInside'`